tools/travis-linux.sh

```diff
@@ -43,7 +43,16 @@ set_up_tests() {
 run_tests() {
     npm run test-selenium
     npm run test-integration-travis
+
     # Note: Cargo recompiles every dependency with dead code. That's why
     # this step is currently the last
+
+    echo "run_tests: Running crate tests..."
```

Comment:
FYI: I [centralized that logic](https://github.com/fxbox/foxbox/pull/406/files#diff-f4983c80cbd9b1e076492899a26bd4d8R11) in #406. 

As rebasing #406 might be tricky (due to commits coming from another repo), would you mind waiting on that PR to land?